harvest/experts-client/bin/experts-grant-feed.js

```diff
@@ -26,19 +22,37 @@ const __filename = fileURLToPath(import.meta.url);
 const __dirname = dirname(__filename);
 
 
+// This also reads data from .env file via dotenv
+const fuseki = new FusekiClient({
+  url: process.env.EXPERTS_FUSEKI_URL || 'http://localhost:3030',
+  auth: process.env.EXPERTS_FUSEKI_AUTH || 'admin:testing123',
+  type: 'tdb2',
+  replace: true,
+});
+
 program
   .version('1.0.0')
   .description('Upload a file to a remote SFTP server')
   .option('--env <env>', '', 'QA')
   .requiredOption('-xml, --xml <xml>', 'Source file path in GCS')
   .requiredOption('-o, --output <output>', 'Local output file path')
   .option('--upload', 'Upload the file to the SFTP server')
-  .option('--fuseki <db>', 'Fuseki database name', 'ae-grants')
+  .option('--fuseki.type <type>', 'specify type on dataset creation', fuseki.type)
```

Comment:
everything but the --delete part can be removed.

Reply:
. Not sure I understand this particular edit.